### PR TITLE
Add annotations to show ocs disk status on UI for LSO cluster

### PR DESCRIPTION
**Pull request by prsurve**
Fixes: #3220 

Signed-off-by: Pratik Surve <prsurve@redhat.com>

### CODE REVIEW

**Review by ebenahar — APPROVED** [review]

**Review by wusui — APPROVED** [review]

**Review by fbalak — DISMISSED** [review]

File: ocs_ci/deployment/deployment.py [hunk 1]
```diff
@@ -508,6 +508,7 @@ def deploy_ocs_via_operator(self):
                 ] = self.DEFAULT_STORAGECLASS
 
             ocs_version = float(config.ENV_DATA['ocs_version'])
+            ocp_version = get_ocp_version()
```
Comment (petr-balogh) [inline ocs_ci/deployment/deployment.py:511]: I know we discuss it some time back that string comparation also works, but can we use float here as well?
```suggestion
            ocp_version = float(get_ocp_version())
```

File: ocs_ci/deployment/deployment.py [hunk 2]
```diff
@@ -529,6 +530,10 @@ def deploy_ocs_via_operator(self):
                             'memory': '5Gi'
                         }
                     }
+                if (ocp_version >= "4.6") and (ocs_version >= 4.6):
```
Comment (petr-balogh) [inline ocs_ci/deployment/deployment.py:533]: and then 
```suggestion
                if (ocp_version >= 4.6) and (ocs_version >= 4.6):
```

File: ocs_ci/deployment/deployment.py (hunk 1 above)
Comment (prsurve) [reply ocs_ci/deployment/deployment.py]: Done

File: ocs_ci/deployment/deployment.py (hunk 2 above)
Comment (prsurve) [reply ocs_ci/deployment/deployment.py]: Done

File: ocs_ci/deployment/deployment.py [hunk 3]
```diff
@@ -990,7 +995,7 @@ def setup_local_storage(storageclass):
         f"{constants.OPERATOR_NODE_LABEL}"
     )
 
-    ocp_version = get_ocp_version()
+    ocp_version = float(get_ocp_version())
```
Comment (petr-balogh) [inline ocs_ci/deployment/deployment.py:998]: This is problematic. As on line 1087 you comparing string and float now.
https://github.com/red-hat-storage/ocs-ci/pull/3221/files#diff-8571fc52f1040ac2cb01985da6d61cc93a535f812814c200b45d00daf28c2e3fR1087

So either do not change it here, or change all usage for compare with this variable.

Comment (prsurve) [reply ocs_ci/deployment/deployment.py:998]: Sorry looks like i changed the wrong line

**Review by petr-balogh — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Pull request merged**